scripts/apps/search/views/save-search-dialog.html

```diff
@@ -1,6 +1,6 @@
 <!-- <form name="viewsForm"> -->
     <fieldset>
-        <div class="field" ng-if="edit.filter.query.raw">
+        <div class="field" ng-hide="{{edit.filter.query.raw === undefined}}">
```

Comment:
`== null` covers for both `null` and `undefined`. Use `== null` if there is no special meaning when the value is null.

Reply:
I have to hide it in case of undefined only, that's why I didn't use `== null`